distributedcombigrid/src/sgpp/distributedcombigrid/manager/ProcessManager.cpp

```diff
@@ -396,6 +396,29 @@ std::vector<double> ProcessManager::evalErrorOnDFG(const LevelVector& leval, siz
   return g->evalErrorOnDFG(leval);
 }
 
+std::vector<CombiDataType> ProcessManager::interpolateValues(std::vector<std::vector<real>>& interpolationCoords) {
+  auto numValues = interpolationCoords.size();
+  std::vector<std::vector<CombiDataType>> values (pgroups_.size(), std::vector<CombiDataType>(numValues));
+  std::vector<MPI_Request> requests(pgroups_.size());
+  for (size_t i = 0; i < pgroups_.size(); ++i) {
+    pgroups_[i]->interpolateValues(interpolationCoords, values[i], &requests[i]);
+  }
+  // // these here don't really wait, for some reason I don't understand right now...
+  // MPI_Wait(&requests[0], MPI_STATUS_IGNORE);
```

Comment:
@obersteiner maybe you can see what is wrong with this waitall call?
the requests should be filled by the call to pgroups_[i]->interpolateValues(...) ...